src/WebUI/Utils.ts

```diff
@@ -3,10 +3,22 @@
     Licensed under the MIT license.
 */
 
-import { V1ObjectMeta } from "@kubernetes/client-node";
+import { V1ObjectMeta, V1PodStatus } from "@kubernetes/client-node";
 import { format } from "@uifabric/utilities/lib";
 import { ObservableArray } from "azure-devops-ui/Core/Observable";
 import { ILabelModel } from "azure-devops-ui/Label";
+import { Statuses, IStatusProps } from "azure-devops-ui/Status";
+
+/**
+ * https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase
+ */
+export enum PodPhase {

```

Comment:
if ant used any where we can make this as without export, if used, we can keep this in types.ts file